src/node_api.cc

```diff
@@ -570,8 +570,7 @@ class SetterCallbackWrapper
 
   /*virtual*/
   void SetReturnValue(napi_value value) override {
-    node::FatalError("napi_set_return_value",
-      "Cannot return a value from a setter callback.");
+    assert(false);  // Cannot return a value from a setter callback
```

Comment:
I actually had the code copied earlier, when I got the "bright" idea to forward-declare. Lemme test that on both Linux, Windows, and OSX.